scripts/train_fixtures.py

```diff
@@ -13,7 +13,6 @@
 from allennlp.commands.train import train_model_from_file, train_model
```

Comment:
I simplified this file a bit because most of the models have moved.  It looks like a corresponding script doesn't exist in the models repo yet, though.  Might be nice to move this over at some point.  Though I suppose if someone wants this, they can just find the old version of this file from a prior version, and copy it over when they need it...